apps/web/app/api/checkNftProof/route.ts

```diff
@@ -0,0 +1,31 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { Address } from 'viem';
+import { getKv } from 'apps/web/src/utils/datastores/kv';
+import { logger } from 'apps/web/src/utils/logger';
+
+type RequestBody = {
+  address: Address;
+};
+
+export async function GET(request: NextRequest) {
```

Comment:
issue: Should this be a POST or use query params to get the address? You're parsing the body below and GET requests shouldn't have bodies.

Reply:
yes, should be a post. fixed